backend-rust/src/migrations/m00010-ranking-by-total-stake.sql

```diff
@@ -0,0 +1,11 @@
+-- Rename table and colum since the table is re-created with new data for each block and not only for each payday block now.
+ALTER TABLE bakers_payday_lottery_powers 
+RENAME TO bakers_lottery_powers;
+ALTER TABLE bakers_lottery_powers
+RENAME COLUMN payday_lottery_power TO lottery_power;
+
+-- Ranking all bakers by lottery powers staring with rank 1 for the baker with the highest lottery power.
```

Comment:
```suggestion
-- Ranking all bakers by lottery powers
```
We can order both ASC/DESC, so talking about the ordering is a redundant/misleading.

Reply:
Tbh I would prefer just removing the comment, since the name of the column is self explanatory.

Reply:
hmm, I don't think the order is self-explanatory. A ranking can go from highest to lowest in terms of lottery power or from lowest to highest.